modules/cnv-initiating-vm-migration-web.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-migrate-vmi.adoc
+
+[id="cnv-initiating-vm-migration-web_{context}"]
+= Initiating live migration of a virtual machine instance in the web console
+
+Migrate a running virtual machine instance to a different node in the cluster. 
+
+Migrate a virtual machine using the &#8942; button of the template in the 
+*Workloads* -> *Virtual Machines* list, or using the *Actions* control of the 
+*Virtual Machine Details* screen.
+
+[NOTE]
+====
+Only admin users can initiate a virtual machine migration, although the 
+*Migrate Virtual Machine* action is visible to all users.
+====
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
+. Click the &#8942; button of the virtual machine to migrate and select 
```

Comment:
My suggestions for these alternate paths are the same as in the other module: I'd pull one, but if you keep them both, please restructure the step.